### DISCUSSION TITLE
Is there any support for ByteArray?

### DISCUSSION THREAD

**Discussion by shadikkhan** [Netflix/dgs-framework#1159] [Q&A] [answered]
I have REST API which is returning ByteArray (an image). now I am converting that to graphql using DGS but so far did not figure out how to handle this byte array response.

Any help is highly appreciated. Thank

**Comment by srinivasankavitha** [comment, marked as answer]
We don't have any special support built into the framework for byte arrays. However, you can implement a custom scalar, e.g. `ByteArray` and specify how it should be serialized/deserialized: https://netflix.github.io/dgs/scalars/
That could work in your case.
